core/templates/dev/head/pages/exploration_editor/translation_tab/TranslatorOverviewDirective.js

```diff
@@ -79,6 +79,22 @@ oppia.directive('translatorOverview', [
             $window.localStorage.setItem(
               LAST_SELECTED_TRANSLATION_LANGUAGE, $scope.languageCode);
           };
+
+          $scope.getTranslationProgressAriaLabel = function() {
+            if ($scope.numberOfRequiredAudio -
+              $scope.numberOfAudioNotAvailable < 1) {
```

Comment:
Change this condition to === 1. Even for 0, it should be items.

Reply:
Done